test/markdownlint-test-result-object.js

```diff
@@ -346,7 +346,13 @@ test.cb("resultFormattingV3", (t) => {
         "# Heading   \n" +
         "\n" +
         "Text\ttext\t\ttext\n" +
-        "Text * emphasis * text"
+        "Text * emphasis * text\n" +
```

Comment:
Could the changes to this file be reverted by adding a third set of test files that leave the default value of 1?